shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationAttempts.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.di.authentication.shared.entity;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class AuthenticationAttempts {
+
+    private String attemptIdentifier;
+    private String email;
+    private String journeyType;
+    private String authenticationType;
+    private String code;
+    private Integer count;
+    private long timeToLive;
+    private String created;
+    private String updated;
+
+    @DynamoDbPartitionKey
+    @DynamoDbAttribute("AttemptIdentifier")
+    public String getAttemptIdentifier() {
+        return attemptIdentifier;
+    }
+
+    public void setAttemptIdentifier(String attemptIdentifier) {
+        this.attemptIdentifier = attemptIdentifier;
+    }
+
+    public AuthenticationAttempts withAttemptIdentifier(String attemptIdentifier) {
+        this.attemptIdentifier = attemptIdentifier;
+        return this;
+    }
+
+    @DynamoDbAttribute("Email")
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    @DynamoDbAttribute("JourneyType")
+    public String getJourneyType() {
+        return journeyType;
+    }
+
+    public void setJourneyType(String journeyType) {
+        this.journeyType = journeyType;
+    }
+
+    @DynamoDbAttribute("AuthenticationType")
+    public String getAuthenticationType() {
+        return authenticationType;
+    }
+
+    public void setAuthenticationType(String authenticationType) {
```

Comment:
Some fields have builder pattern `withX()` methods, and some don't (like `journeyType` and `authenticationType`.

Can we add them to all missing fields? Makes me wonder if we can remove `set*()` methods too.